src/CommitmentsV2/SFA.DAS.CommitmentsV2.UnitTests/Application/Queries/GetProvider/GetProviderQueryHandlerTests.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using NUnit.Framework;
+using SFA.DAS.CommitmentsV2.Application.Queries.GetProvider;
+using SFA.DAS.CommitmentsV2.Data;
+using SFA.DAS.CommitmentsV2.Models;
+
+namespace SFA.DAS.CommitmentsV2.UnitTests.Application.Queries.GetProvider
+{
+    [TestFixture]
+    [Parallelizable]
+    public class GetProviderQueryHandlerTests
```

Comment:
There is a danger here of failing tests, due to the Parallelizable attribute

see this article
https://github.com/nunit/nunit/issues/2105

Options are to remove the Parallelizable attribute or instantiate the fixture per test. Both options are good.

Reply:
After discussion with Ian and Paul, this is not an issue. The parallelizable attribute on the class only makes the class run in parallel with other parallelizable classes. It doesn't make the elements within class run in parallel with each other, so this is not an issue.